lib/msf/core/modules/metadata/obj.rb

```diff
@@ -80,6 +84,12 @@ def initialize(module_instance, obj_hash = nil)
     @path               = module_instance.file_path
     @mod_time           = ::File.mtime(@path) rescue Time.now
     @ref_name           = module_instance.refname
+    if module_instance.respond_to?('autofilter_ports')
```

Comment:
Could the symbol `:autofilter_ports` be used here instead of a string? (See also `autofilter_services` below)